src/components/Toast/Toast.module.css

```diff
@@ -10,6 +10,7 @@
   position: fixed;
   top: -15%;
   left: 50%;
+  z-index: 9000;
```

Comment:
Should we discuss later a structure to have an organize way to implement the z-index? I feel like sometimes we just play around and add a large number, and that could cause bugs for future developers.

Reply:
Totally agree!